wireguard-install.sh

```diff
@@ -12,17 +12,32 @@ function isRoot() {
 
 function checkVirt() {
 	if [ "$(systemd-detect-virt)" == "openvz" ]; then
-		echo "OpenVZ is not supported"
-		exit 1
+		if ip link add wg999 type wireguard 2> /dev/null ; then
+			echo "OpenVZ is not supported, but it seems to have correct kernel modules."
+			ip link del wg999
+			read -rp "Press enter to continue at your own risk, or CTRL-C to quit."
+			Container=1
```

Comment:
For consistency, I'd rather use `$CONTAINER` or `$CT`

Reply:
Sounds good :)